docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -5,15 +5,33 @@
 - [Install necessary prerequisites]($docs/getting-started/development-prerequisites)
 - [Clone desktop-starter repo](https://github.com/imodeljs/desktop-starter)
 
+## Create client
+
+- Navigate to [developer.bentley.com](https://developer.bentley.com/register/)

```

Comment:
The term "client" doesn't appear anywhere on the linked page. Are you using it as a synonym for "application"?

Reply:
Hmm yes. I do seem to be mixing terms between client and application. I'll settle on Application for this portion.